app/MMB/src/scenes/timeline/Events.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import * as React from 'react';
+import { DateFormatter, DateUtils } from '@kiwicom/mobile-localization';
+import { View } from 'react-native';
+
+import Event, { type Props as EventType } from './event/Event';
+import Day from './event/Day';
+
+type Props = {|
+  events: Array<EventType>,
```

Comment:
Our prop type should be readonly `+events:`

Reply:
It's odd to import `Prop` type from the component just to prop-drill the props. We should compose the components instead. Why not to expect `Event` components here directly?